backend/de.metas.manufacturing/src/main/sql/postgresql/ddl/functions/PP_Product_BOM_Recursive.sql

```diff
@@ -1,105 +1,100 @@
 drop function if exists PP_Product_BOM_Recursive(numeric, varchar);
-create or replace function PP_Product_BOM_Recursive(p_PP_Product_BOM_ID numeric, p_ad_language varchar)
-returns table
-(
-	Line text,
-	Parent_Line text,
-	ProductValue varchar,
-	ProductName varchar,
-	QtyBOM numeric,
-	Percentage numeric,
-	UOMSymbol varchar,
-	--
-	depth integer,
-	M_Product_ID numeric,
-	IsQtyPercentage char(1),
-	C_UOM_ID numeric,
-    path integer[]
+CREATE OR REPLACE FUNCTION pp_product_bom_recursive(p_pp_product_bom_id numeric, p_ad_language character varying)
+    RETURNS TABLE(line text, parent_line text, productvalue character varying, productname character varying, qtybom numeric, percentage numeric, uomsymbol character varying, depth integer, m_product_id numeric, isqtypercentage character, c_uom_id numeric, path integer[], supplier text)
```

Comment:
@mi-ps  pls correct the code format for `returns table` to one column per line (so we will be able to easily review it in the future)